### ISSUE TITLE
New privacy like function - Sound on / off - Pine + F9?

### ISSUE THREAD

**Issue by Lady-Galadriel** [jackhumbert/pinebook-pro-keyboard-updater#7]
There are times when you want to disable sound semi-permanently. Perhaps a new combination similar to the privacy switches could be made. It would be stored like the others. And when toggled, it would send similar key stroke like you have for the others. Maybe F20 & F21, (if those are valid key events).

Software would have to take advantage of it. So the default would just ignore the event.

**Comment by jackhumbert** [comment]
F20 and F21 could work for that - is there any reason you wouldn't want to use the audio mute functionality that is already there via Fn + F4? It'd be nice if the keyboard controller had access to the amplifier like the other cuts.

**Comment by Lady-Galadriel** [comment]
Yes, Fn +F4 can probably work. I was thinking about a way to over-ride the user panel for audio. Any new user problably has audio re-enabled.

Perhaps if we had a way to get the status of the 3 privacy switches and any additional psuedo-switches, (like this audio), we can perform some over-ride of the user's configuration from an OS startup script. (Not for the 3 privacy switches, we would only be able to get status of them.)

That brings forth an idea, which I will put in a separate issue, having `lsusb -v` return the status of the privary switches.
